src/modules/components/timeline/timeline/timeline.demo.html

```diff
@@ -21,9 +21,22 @@ <h3 class="bold-small">Charging advice requested</h3>
                 <time class="font-xsmall" datetime="09:30Z">9:30am</time>
               </div>
             </article>
+            <br>
+            <header class="timeline-header">
+              <h2 class="bold-medium">Timeline without stem</h2>
+            </header>
+            <article class="timeline-block timeline-block--no-stem">
+              <div class="timeline-block-header">
+                <time datetime="2016-05-10">10 May 2016</time>
+              </div>
+              <div class="timeline-block-content">
+                <h3 class="bold-small">Charging advice requested</h3>
+                <time class="font-xsmall" datetime="09:30Z">9:30am</time>
+              </div>
+            </article>
```

Comment:
It would be nice really to have this as one complete component, rather than timeline specific?

Reply:
Yeah, probably, though PDK for angular 1.x has no-one really maintaining it and many of its components are legacy implementations (an updated PDK for angular 2+ is being developed).

If you want to do a PR, we can merge it though!